httpd/handler.go

```diff
@@ -248,7 +248,7 @@ func (h *Handler) serveStatus(w http.ResponseWriter, r *http.Request) {
 	} else {
 		b, _ = json.Marshal(data)
 	}
-	w.Write(b)
+	_, _ = w.Write(b)
```

Comment:
I'm not sure I follow you.  Do you mean "can't" or "shouldn't"?  I haven't tried it, but it feels like based on the documentation we can pass it the handler.go file and tell it to ignore error checks for `w.Write`.

I think for sure I'm having a very allergic reaction to this :-).